### ISSUE TITLE
Unable to export Provisioning Profiles (Visual Studio Mac Community Edition)

### ISSUE THREAD

**Issue by fatalaa** [bitrise-tools/codesigndoc#50]
```
codesigndoc version: 1.3.0
 => Running codesigndoc scan ...

INFO[14:48:25] Export output dir already exists at path: /Users/timolnar/Desktop/codesigndoc_exports

Please drag-and-drop your Xamarin Solution (.sln)
   file here, and then hit Enter : /Users/timolnar/Development/PrivateStuff/Cockpit365/Cockpit365.sln

Select the Project Name you use for "Archive for Publishing" (usually ends with ".iOS", e.g.: MyProject.iOS)?
Please select from the list:
[1] : Cockpit365.iOS
[2] : Cockpit365.Mac
(type in the option's number, then hit Enter) : 1


INFO[14:48:39] 🔦  Running a Build, to get all the required code signing settings...
INFO[14:48:39] $ "/Library/Frameworks/Mono.framework/Versions/Current/Commands/msbuild" "/Users/timolnar/Development/PrivateStuff/Cockpit365/Cockpit365.sln" "/p:Configuration=Release" "/p:Platform=iPhone" "/t:Cockpit365_iOS"
Running and analyzing log ...............................................................................................................................

INFO[14:50:44]   💡  Saving xamarin output into file: /Users/timolnar/Desktop/codesigndoc_exports/xamarin-build-output.log


=== Required Identities/Certificates (1) ===
 * (1): iPhone Distribution: Michael Amann-Langeder (DHL6EU92D7)
============================================

=== Required Provisioning Profiles (1) ===
 * (1): Cockpit 365 iOS Adhoc (UUID: 476a42d5-bcdd-4dfa-8d27-32884906302e)
==========================================

=== Team IDs (1) ===
 * (1): DHL6EU92D7
==========================================

=== App/Bundle IDs (1) ===
 * (1): DHL6EU92D7.com.cockpit365.ios
==========================================

Do you want to export these files? [YES/no]: YES

INFO[14:56:46] Searching for required Provisioning Profiles ...

INFO[14:56:46]  * Searching for required Provisioning Profile: Cockpit 365 iOS Adhoc (UUID: 476a42d5-bcdd-4dfa-8d27-32884906302e)

------------------------------
First of all please make sure that you can Archive your app from Xamarin Studio.
codesigndoc only works if you can archive your app from Xamarin Studio.
If you can, and you get a valid IPA file if you export from Xamarin Studio,
please create an issue on GitHub at: https://github.com/bitrise-tools/codesigndoc/issues
with as many details & logs as you can share!
------------------------------

Error: Failed to export Provisioning Profiles, error: Failed to find Provisioning Profile: Failed to find Provisioning Profile with UUID: 476a42d5-bcdd-4dfa-8d27-32884906302e
```

_In addition to this, if `codesigndoc` generated an Xcode output log, please attach that too._
[xamarin-build-output.txt](https://github.com/bitrise-tools/codesigndoc/files/1208136/xamarin-build-output.txt)

**Comment by viktorbenei** [comment]
Did you try to "Archive for Publishing" in Xamarin Studio/Visual Studio for Mac with **"Cockpit365.iOS"**?

Seems the provisioning profile set for that project does not exist on your Mac (`Cockpit 365 iOS Adhoc (UUID: 476a42d5-bcdd-4dfa-8d27-32884906302e)`)

**Comment by fatalaa** [comment]
Yes, I was able to export the IPA to my disk.
On Aug 8, 2017, at 3:26 PM, Viktor Benei <notifications@github.com<mailto:notifications@github.com>> wrote:


Did you try to "Archive for Publishing" in Xamarin Studio/Visual Studio for Mac with "Cockpit365.iOS"?

Seems the provisioning profile set for that project does not exist on your Mac (Cockpit 365 iOS Adhoc (UUID: 476a42d5-bcdd-4dfa-8d27-32884906302e))

**Comment by viktorbenei** [comment]
And does that Prov Profile exist at `$HOME/Library/MobileDevice/Provisioning Profiles`?

**Comment by viktorbenei** [comment]
By default it should exist with the UUID in its name; can that be the issue? Maybe you moved the file into Prov Profiles by hand instead of opening it in Xcode which would have placed it as `476a42d5-bcdd-4dfa-8d27-32884906302e.mobileprovision` in that directory?..

https://github.com/bitrise-tools/codesigndoc/blob/master/provprofile/provprofile.go#L159

**Comment by fatalaa** [comment]
Yes, I moved it to to `$HOME/Library/MobileDevice/Provisioning Profiles` by the `cp` command.
Installing via `Xcode` resolved the issue.
Thanks.

**Comment by viktorbenei** [comment]
Glad to hear it worked ;)

That said, I'll reopen this issue and we'll roll out a fix for this - if Xcode/Xamarin Studio can work with that file `codesigndoc` should too. Actually most of the searchers ignore the file name, except the one I linked, will add content search there too.

**Comment by viktorbenei** [comment]
Fixed in #51, it should now find the mobile prov profile even if the name of the profile is not the UUID (e.g. when moved manually into the prov profiles dir).

Thanks again @fatalaa for reporting! ;)
